Conference-Room-Reservation-System/Conference-Room-Reservation-System/src/test/java/com/sda/conferenceroomreservationsystem/repository/OrganizationRepositoryTest.java

```diff
@@ -0,0 +1,48 @@
+package com.sda.conferenceroomreservationsystem.repository;
+
+import com.sda.conferenceroomreservationsystem.model.entity.Organization;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.Optional;
+
+
+@DataJpaTest
+class OrganizationRepositoryTest {
+
+    @Autowired
+    private OrganizationRepository organizationRepository;
+
+    @Test
+    void findByOrganizationNameShouldReturnOrganizationEntity() {
+        String organizationName = "Transporeon";
+
+        Organization organization = new Organization();
+        organization.setOrganizationName(organizationName);
```

Comment:
Consider creating method which will save entity in database instead of doubling code in two tests